src/test/java/io/ebean/dbmigration/ddlgeneration/platform/PlatformDdl_AlterColumnTest.java

```diff
@@ -157,7 +162,7 @@ public void testAlterColumnDefaultValue() throws Exception {
     assertEquals("alter table mytab alter acol set default 'hi'", sql);
 
     sql = sqlServerDdl.alterColumnDefaultValue("mytab", "acol", "'hi'");
-    assertEquals("alter table mytab add constraint df_mytab_acol default 'hi' for acol", sql);
```

Comment:
I had problems with named default constrainsts & temporal tables in SqlServer. (I could not add non null columns) - so I reverted the change for named columns and let the sqlserver generate a (random) name for defaults.

Reply:
ok